R/renv.R

```diff
@@ -282,3 +282,42 @@ renv_switch_r_version <- function(version = NULL
 
   return(invisible(TRUE))
 }
+
+#' @title Build a local package and install it into an renv project
+#' @description This is a wrapper around `pkgbuild::build()` and
+#'   `renv::install()` to more easily make local packages available within
+#'   \pkg{renv} projects.
+#'
+#' The following steps are performed:
+#'
+#' 1. Building the package found at argument `path` via `pkgbuild::build()`.
+#' 2. Moving the built source into the \pkg{renv} cache. The cache location is
+#' determined by `Sys.getenv("RENV_PATHS_LOCAL")`.
+#' 3. Installing the package from the cache location via `renv::install()`.
+#' @param path `[character]`\cr
+#'   The path to the package which should be built and installed.
+#' @importFrom renv install
+#' @export
+renv_install_local <- function(path = ".") {
+  if (path == ".") {
+    path <- usethis::proj_get()
+  }
+  if (Sys.getenv("RENV_PATHS_LOCAL") == "") {
```

Comment:
```suggestion
  if (renv_local == "") {
```

and move the assignment up?